vane/tests_tools.py

```diff
@@ -268,6 +268,8 @@ def login_duts(test_parameters, test_duts):
         login_ptr["mgmt_ip"] = dut["mgmt_ip"]
         login_ptr["username"] = dut["username"]
         login_ptr["password"] = dut["password"]
+        if "role" not in dut:
+            dut["role"] = ""
         login_ptr["role"] = dut["role"]
```

Comment:
Rather than assigning value to the dut dictionary then reassigning to the login_ptr dictionary, maybe just use the dictionary.get() method with a default of an empty string.

```suggestion
        login_ptr["role"] = dut.get("role", "")
```

Reply:
@grybak-arista thanks for the suggestion, should be good now?

Reply:
Also @grybak-arista this PR is from when coverage and unittests were under same workflow "unittest" so after you resolve this conversation, you should be good to merge it (will have to bypass the workflow)👍🏻

Reply:
perfect!